### PR TITLE
typo fix and pronoun tagging

### PR DESCRIPTION
**Pull request by selkirks**
fixed a typo in new_cat_hostile, pronoun tagged other_clan_hostile, and fixed a pronoun tag in plains hunting leaf_fall

### CODE REVIEW

File: resources/dicts/patrols/other_clan_hostile.json [hunk 1]
```diff
@@ -679,7 +679,7 @@
                 "other_clan_rep": -1
             },
             {
-                "text": "r_c is having trouble focusing today, and puts a mark in the wrong place - and at the worst possible moment, because a o_c_n patrol sees them doing it! They come back to camp battered and scratched, and immediately apologize to the deputy for the diplomatic incident they've inadvertently caused.",
+                "text": "r_c is having trouble focusing today, and puts a mark in the wrong place - and at the worst possible moment, because a o_c_n patrol sees them doing it! {PRONOUN/r_c/subject/CAP} come back to camp battered and scratched, and immediately apologize to the deputy for the diplomatic incident {PRONOUN/r_c/subject}{VERB/r_c/'re/'s} inadvertently caused.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: Think this should be {PRONOUN/r_c/subject}{VERB/r_c/'ve/'s}

File: resources/dicts/patrols/other_clan_hostile.json [hunk 2]
```diff
@@ -804,13 +804,13 @@
...
                 "exp": 0,
                 "weight": 20,
                 "other_clan_rep": -1
             },
             {
-                "text": "s_c is marking along the territory line when they encounter a patrol from o_c_n, out on a similar mission. They stare at the other Clan's patrol challengingly, following their every move until the shared border ends, and the patrol is marred by muttered insults and hissed threats from both sides.",
+                "text": "s_c is marking along the territory line when {PRONOUN/s_c/subject} encounter a patrol from o_c_n, out on a similar mission. {PRONOUN/s_c/subject/CAP} stare at the other Clan's patrol challengingly, following their every move until the shared border ends, and the patrol is marred by muttered insults and hissed threats from both sides.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "encounter" and "stare"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 3]
```diff
@@ -778,14 +778,14 @@
                 "other_clan_rep": 2
             },
             {
-                "text": "s_c is marking along the territory line when they encounter a patrol from o_c_n, out on a similar mission. They stare at the other Clan's patrol challengingly, following their every move until the shared border ends. But by the standards of o_c_n - c_n relations, it's a peaceful encounter.",
+                "text": "s_c is marking along the territory line when {PRONOUN/s_c/subject} encounter a patrol from o_c_n, out on a similar mission. {PRONOUN/s_c/subject/CAP} stare at the other Clan's patrol challengingly, following their every move until the shared border ends. But by the standards of o_c_n - c_n relations, it's a peaceful encounter.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "encounter" and "stare"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 4]
```diff
@@ -293,7 +293,7 @@
                 "other_clan_rep": -1
             },
             {
-                "text": "r_c checks the area, it seems to be clear of o_c_n patrols. They leap for the rabbit, easily killing it, but very quickly find that they should've double checked before trespassing. A o_c_n patrol rushes into view from where they were hiding and descends upon r_c, killing them before anyone else can help. Was one rabbit worth this?",
+                "text": "r_c checks the area, it seems to be clear of o_c_n patrols. {PRONOUN/r_c/subject/CAP} leap for the rabbit, easily killing it, but very quickly find that {PRONOUN/r_c/subject} should've double checked before trespassing. A o_c_n patrol rushes into view from where they were hiding and descends upon r_c, killing {PRONOUN/r_c/object} before anyone else can help. Was one rabbit worth this?",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "leaps"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 5]
```diff
@@ -356,26 +356,26 @@
...
         },
         "weight": 20,
-        "intro_text": "p_l pads along, mouth open slightly as they scent for any traces of herbs. Oh! They can smell some raspberries nearby - but it seems that they're just over the border.",
+        "intro_text": "p_l pads along, mouth open slightly as {PRONOUN/p_l/subject} scent for any traces of herbs. Oh! {PRONOUN/r_c/subject/CAP} can smell some raspberries nearby - but it seems that they're just over the border.",
         "decline_text": "They decide that trespassing isn't worth it in this case.",
         "chance_of_success": 30,
         "success_outcomes": [
             {
-                "text": "They pad over to the border and hesitate only momentarily before crossing. They quickly gather some of the berries, making sure to leave plenty for o_c_n's medicine cat if they need some, too.",
+                "text": "{PRONOUN/p_l/subject/CAP} pad over to the border and hesitate only momentarily before crossing. {PRONOUN/p_l/subject/CAP} quickly gather some of the berries, making sure to leave plenty for o_c_n's medicine cat if they need some, too.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "pads" and "gathers"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 6]
```diff
@@ -356,26 +356,26 @@
...
                 "exp": 20,
                 "weight": 20,
                 "herbs": ["raspberry"],
                 "other_clan_rep": 2
             },
             {
-                "text": "They pad to the border and pause for a heartbeat to scent the air. o_c_n's scent is very strong, but there's something else... o_c_n's medicine cat appears at that moment. They're walking towards the raspberries when they spot p_l and curl their lip. They snarl that they're not willing to share with an enemy Clan. p_l takes the hint and leaves.",
+                "text": "{PRONOUN/p_l/subject/CAP} pad to the border and pause for a heartbeat to scent the air. o_c_n's scent is very strong, but there's something else... o_c_n's medicine cat appears at that moment. They're walking towards the raspberries when they spot p_l and curl their lip. They snarl that they're not willing to share with an enemy Clan. p_l takes the hint and leaves.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "pads"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 7]
```diff
@@ -356,26 +356,26 @@
...
                 "exp": 20,
                 "weight": 5,
                 "herbs": ["raspberry"],
                 "other_clan_rep": 2
             },
             {
-                "text": "p_l is about to step over the border when a hiss stops them in their tracks. A o_c_n warrior bounds into view, bristling with rage. They quickly explain that they're the medicine cat and the warrior code allows them to cross the border and assure that they only need a few berries for some sick Clanmates. The warrior looks torn, but agrees to this.",
+                "text": "p_l is about to step over the border when a hiss stops {PRONOUN/p_l/object} in {PRONOUN/p_l/poss} tracks. A o_c_n warrior bounds into view, bristling with rage. {PRONOUN/p_l/subject/CAP} quickly explain that {PRONOUN/p_l/subject}{VERB/r_c/'re/'s} the medicine cat and the warrior code allows {PRONOUN/p_l/object} to cross the border and assure that {PRONOUN/p_l/subject} only need a few berries for some sick Clanmates. The warrior looks torn, but agrees to this.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "explains" and "needs"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 8]
```diff
@@ -385,13 +385,13 @@
         ],
         "fail_outcomes": [
             {
-                "text": "p_l starts to step over the border but a hiss stops them in their tracks. A o_c_n patrol is running their way. As the patrol approaches, they blurt out that they're a medicine cat. One of the warriors snarls that they won't tolerate c_n cats on their land, medicine cat or not, and p_l needs to leave before they make them. They oblige quickly.",
+                "text": "p_l starts to step over the border but a hiss stops {PRONOUN/p_l/object} in {PRONOUN/p_l/poss} tracks. A o_c_n patrol is running {PRONOUN/p_l/poss} way. As the patrol approaches, {PRONOUN/p_l/subject} blurt out that {PRONOUN/p_l/subject}{VERB/r_c/'re/'s} a medicine cat. One of the warriors snarls that they won't tolerate c_n cats on their land, medicine cat or not, and p_l needs to leave before they make {PRONOUN/p_l/object}. {PRONOUN/p_l/subject/CAP} oblige quickly.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "blurts" and "obliges"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 9]
```diff
@@ -385,13 +385,13 @@
...
                 "exp": 0,
                 "weight": 20,
                 "other_clan_rep": -1
             },
             {
-                "text": "p_l starts to step over the border but a hiss stops them in their tracks. A o_c_n patrol is running their way. As the patrol approaches, they blurt out that they're a medicine cat. The warriors either don't hear them or don't care and they take multiple swipes at the medicine cat before they can get away.",
+                "text": "p_l starts to step over the border but a hiss stops {PRONOUN/p_l/object} in {PRONOUN/p_l/poss} tracks. A o_c_n patrol is running {PRONOUN/p_l/poss} way. As the patrol approaches, {PRONOUN/p_l/subject} blurt out that {PRONOUN/p_l/subject}{VERB/r_c/'re/'s} a medicine cat. The warriors either don't hear {PRONOUN/p_l/object} or don't care and they take multiple swipes at the medicine cat before {PRONOUN/p_l/subject} can get away.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "blurts"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 10]
```diff
@@ -519,7 +519,7 @@
         "chance_of_success": 30,
         "success_outcomes": [
             {
-                "text": "After poking around a bit, p_l discovers a o_c_n apprentice wandering nearby, hopelessly lost. The apprentice is distraught upon discovering they're on the wrong side of the border and while p_l is still suspicious of the situation, they only lecture the apprentice on border lines before sending them back into o_c_n territory.",
+                "text": "After poking around a bit, p_l discovers a o_c_n apprentice wandering nearby, hopelessly lost. The apprentice is distraught upon discovering they're on the wrong side of the border and while p_l is still suspicious of the situation, {PRONOUN/p_l/subject} only lecture the apprentice on border lines before sending them back into o_c_n territory.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "lectures"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 11]
```diff
@@ -586,7 +586,7 @@
         ],
         "antag_fail_outcomes": [
             {
-                "text": "How dare they trespass! p_l bounds after the scent, hoping to find the source of it, eventually discovering a limping o_c_n warrior who hastily explains that a dog had chased them into c_n's territory. p_l is still angry, but they rein in their hostility and tensely return the cat to o_c_n territory.",
+                "text": "How dare they trespass! p_l bounds after the scent, hoping to find the source of it, eventually discovering a limping o_c_n warrior who hastily explains that a dog had chased them into c_n's territory. p_l is still angry, but {PRONOUN/p_l/subject} rein in their hostility and tensely return the cat to o_c_n territory.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "reins" and "returns"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 12]
```diff
@@ -631,14 +631,14 @@
                 "other_clan_rep": 2
             },
             {
-                "text": "s_c is marking along the territory line when they encounter a patrol from o_c_n, out on a similar mission. They stare at the other Clan's patrol challengingly, following their every move until the shared border ends. But by the standards of o_c_n - c_n relations, it's a peaceful encounter.",
+                "text": "s_c is marking along the territory line when {PRONOUN/s_c/subject} encounter a patrol from o_c_n, out on a similar mission. {PRONOUN/s_c/subject/CAP} stare at the other Clan's patrol challengingly, following their every move until the shared border ends. But by the standards of o_c_n - c_n relations, it's a peaceful encounter.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "encounters" and "stares"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 13]
```diff
@@ -631,14 +631,14 @@
...
                 "exp": 20,
                 "weight": 20,
                 "stat_skill": ["SPEAKER,1"],
                 "other_clan_rep": 2
             },
             {
-                "text": "As s_c marks the border, they encounter a o_c_n border patrol. It even includes a young apprentice, who starts fluffing up and insulting them. s_c sternly growls back to the apprentice that border fights are no joke, and surprisingly their mentor joins s_c, both warriors teaching the youngster firmly but peacefully.",
+                "text": "As s_c marks the border, {PRONOUN/s_c/subject} encounter a o_c_n border patrol. It even includes a young apprentice, who starts fluffing up and insulting them. s_c sternly growls back to the apprentice that border fights are no joke, and surprisingly their mentor joins s_c, both warriors teaching the youngster firmly but peacefully.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "encounters"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 14]
```diff
@@ -657,13 +657,13 @@
         ],
         "fail_outcomes": [
             {
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json:659]: tag "sees"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 15]
```diff
@@ -778,14 +778,14 @@
...
                 "exp": 20,
                 "weight": 20,
                 "stat_skill": ["SPEAKER,1"],
                 "other_clan_rep": 2
             },
             {
-                "text": "As s_c marks the border, they encounter a o_c_n border patrol. It even includes a young apprentice, who starts fluffing up and insulting them. s_c sternly growls back to the apprentice that border fights are no joke, and surprisingly their mentor joins s_c, both warrior and deputy teaching the youngster firmly but peacefully.",
+                "text": "As s_c marks the border, {PRONOUN/s_c/subject} encounter a o_c_n border patrol. It even includes a young apprentice, who starts fluffing up and insulting {PRONOUN/s_c/object}. s_c sternly growls back to the apprentice that border fights are no joke, and surprisingly their mentor joins s_c, both warrior and deputy teaching the youngster firmly but peacefully.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "encounters"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 16]
```diff
@@ -826,7 +826,7 @@
                 "other_clan_rep": -1
             },
             {
-                "text": "r_c encounters a patrol from o_c_n on the border, and greets them coldly, but there's something different here. Something dangerous. They don't return to camp.",
+                "text": "r_c encounters a patrol from o_c_n on the border, and greets them coldly, but there's something different here. Something dangerous. {PRONOUN/r_c/subject/CAP} don't return to camp.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "don't/doesn't"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 17]
```diff
@@ -878,7 +878,7 @@
                 "other_clan_rep": -1
             },
             {
-                "text": "s_c can't contain their hatred for o_c_n, not even meeting a patrol of theirs at the border as a lone cat. They come back to camp bleeding.",
+                "text": "s_c can't contain {PRONOUN/s_c/poss} hatred for o_c_n, not even meeting a patrol of theirs at the border as a lone cat. {PRONOUN/s_c/subject/CAP} come back to camp bleeding.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: Tag "comes"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 18]
```diff
@@ -937,14 +937,14 @@
                 "other_clan_rep": 2
             },
             {
-                "text": "s_c is marking along the territory line when they encounter a patrol from o_c_n, out on a similar mission. They stare at the other Clan's patrol challengingly, following their every move until the shared border ends. But by the standards of o_c_n - c_n relations, it's a peaceful encounter.",
+                "text": "s_c is marking along the territory line when {PRONOUN/s_c/subject} encounter a patrol from o_c_n, out on a similar mission. {PRONOUN/s_c/subject/CAP} stare at the other Clan's patrol challengingly, following their every move until the shared border ends. But by the standards of o_c_n - c_n relations, it's a peaceful encounter.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "encounters" and "stares"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 19]
```diff
@@ -937,14 +937,14 @@
...
                 "exp": 20,
                 "weight": 20,
                 "stat_skill": ["SPEAKER,1"],
                 "other_clan_rep": 2
             },
             {
-                "text": "As s_c marks the border, they encounter a o_c_n border patrol. It even includes a young apprentice, who starts fluffing up and insulting them. s_c sternly growls back to the apprentice that border fights are no joke, and surprisingly their mentor joins s_c, both warrior and leader teaching the youngster firmly but peacefully.",
+                "text": "As s_c marks the border, {PRONOUN/s_c/subject} encounter a o_c_n border patrol. It even includes a young apprentice, who starts fluffing up and insulting {PRONOUN/s_c/object}. s_c sternly growls back to the apprentice that border fights are no joke, and surprisingly their mentor joins s_c, both warrior and leader teaching the youngster firmly but peacefully.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "encounters"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 20]
```diff
@@ -963,13 +963,13 @@
...
                 "exp": 0,
                 "weight": 20,
                 "other_clan_rep": -1
             },
             {
-                "text": "s_c is marking along the territory line when they encounter a patrol from o_c_n, out on a similar mission. They stare at the other Clan's patrol challengingly, following their every move until the shared border ends, and the patrol is marred by muttered insults and hissed threats from both sides.",
+                "text": "s_c is marking along the territory line when {PRONOUN/s_c/subject} encounter a patrol from o_c_n, out on a similar mission. {PRONOUN/s_c/subject/CAP} stare at the other Clan's patrol challengingly, following their every move until the shared border ends, and the patrol is marred by muttered insults and hissed threats from both sides.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "encounters" and "stares"

**Review by Thlumyn — CHANGES_REQUESTED** [review]
I went through and starting tagging some change requests; there's a lot places where you add subject pronoun tagging and then need to change the corresponding verb to be tagged as well. You do it a few times, but not all of them; we need it so we don't end up with things like "They encounter" turning into "He encounter"

File: resources/dicts/patrols/other_clan_hostile.json (hunk 11 above)
Comment (selkirks) [reply resources/dicts/patrols/other_clan_hostile.json:589]: image
i dont believe rein and return are tagged here? i thought it meant 'they' as the whole patrol

**Comment by selkirks** [comment]
> I went through and starting tagging some change requests; there's a lot places where you add subject pronoun tagging and then need to change the corresponding verb to be tagged as well. You do it a few times, but not all of them; we need it so we don't end up with things like "They encounter" turning into "He encounter"

thank you! ive fixed all the ones you mentioned as well as a few i forgot

File: resources/dicts/patrols/other_clan_hostile.json [hunk 21]
```diff
@@ -586,7 +586,7 @@
         ],
         "antag_fail_outcomes": [
             {
-                "text": "How dare they trespass! p_l bounds after the scent, hoping to find the source of it, eventually discovering a limping o_c_n warrior who hastily explains that a dog had chased them into c_n's territory. p_l is still angry, but they rein in their hostility and tensely return the cat to o_c_n territory.",
+                "text": "How dare they trespass! p_l bounds after the scent, hoping to find the source of it, eventually discovering a limping o_c_n warrior who hastily explains that a dog had chased them into c_n's territory. p_l is still angry, but {PRONOUN/p_l/subject} {VERB/p_l/rein/reins} in their hostility and tensely return the cat to o_c_n territory.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "returns"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 22]
```diff
@@ -657,13 +657,13 @@
         ],
         "fail_outcomes": [
             {
-                "text": "r_c comes across a o_c_n apprentice on the wrong side of the border, and holding prey. They see the young cat off with a furious swipe of their claws, and then bring the prey back to c_n camp to report the incident to the deputy.",
+                "text": "r_c comes across a o_c_n apprentice on the wrong side of the border, and holding prey. {PRONOUN/s_c/subject/CAP} {VERB/s_c/see/sees} the young cat off with a furious swipe of {PRONOUN/r_c/poss} claws, and then bring the prey back to c_n camp to report the incident to the deputy.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "brings"

File: resources/dicts/patrols/other_clan_hostile.json [hunk 23]
```diff
@@ -2836,13 +2836,13 @@
         ],
         "fail_outcomes": [
             {
-                "text": "r_c comes across a o_c_n apprentice on the wrong side of the border, and holding prey. They see the young cat off with a furious swipe of their claws, and then bring the prey back to c_n camp to report the incident to the deputy.",
+                "text": "r_c comes across a o_c_n apprentice on the wrong side of the border, and holding prey. {PRONOUN/r_c/subject/CAP} {VERB/r_c/see/sees} the young cat off with a furious swipe of {PRONOUN/r_c/poss} claws, and then bring the prey back to c_n camp to report the incident to the deputy.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/other_clan_hostile.json]: tag "brings"

**Review by Thlumyn — CHANGES_REQUESTED** [review]

**Review by Thlumyn — APPROVED** [review]

**Pull request merged**